src/stores/SpaceStore.tsx

```diff
@@ -51,6 +54,11 @@ export const UPDATE_SELECTED_SPACE = Symbol("selected-space");
 
 const MAX_SUGGESTED_ROOMS = 20;
 
+const getLastViewedRoomsStorageKey = (spaceId?) => {
```

Comment:
I meant more like it takes a `space?: Room` and then does `space?.roomId || LAST_VIEWED_ROOMS_HOME;` internally, sorry about not being clear

Reply:
Yeah, I just realized that. No worries, I don't seem to be functioning very well today... :sweat_smile: